src/features/surveys/components/SurveyOptionsModal/SurveyOptionsModal.tsx

```diff
@@ -38,6 +38,18 @@ export default function SurveyOptionsModalModal({
             }}
             label={t('surveyOptionsModal.OneQuestionPerStep')}
           />
+          <Toggle
+            isEnabled={surveyOptions.displayTitle}
+            classNames="gap-2 mt-4"
+            testId="one-per-step-toggle"
```

Comment:
Please change `testId` somthing like `display-title-toggle`

Reply:
I just pushed up the following commit link with the change. Thank you for the review and feedback!

Changed testId commit: https://github.com/Ryczko/FormsLab/pull/230/commits/f43cbd1cc47b9002d039e2229db94fe25387dcfe